src/include/dng/peeling.h

```diff
@@ -35,7 +35,7 @@ namespace dng {
 namespace peel {
 
 namespace op {
-enum {
+enum OpTypes {
     UP, DOWN, TOFATHER, TOMOTHER, TOCHILD,
```

Comment:
Yes, only purpose is to get ride off 
`warning: ‘dng::RelationshipGraph’ has a field ‘dng::RelationshipGraph::peeling_ops_’ whose type uses the anonymous namespace [-Wsubobject-linkage]`